website/content/en/docs/provisioner.md

```diff
@@ -47,6 +47,14 @@ spec:
     - key: "karpenter.sh/capacity-type" # If not included, the webhook for the AWS cloud provider will default to on-demand
       operator: In
       values: ["spot", "on-demand"]
+
+  # Resource limits constrain the total size of the cluster.
```

Comment:
```suggestion
  # Resource limits that can be launched by this provisioner.
```

Reply:
I'm confused by this suggestion. It seems to be missing something between "limits" and "that can..."